### PR TITLE
Document Profiling+SSI beta k8s support

### PR DESCRIPTION
**Pull request by szegedi**
### What does this PR do? What is the motivation?
Adds documentation for enabling Profiling+SSI support on Kubernetes.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/profiler/enabling/ssi.md [hunk 1]
```diff
@@ -26,40 +25,104 @@ Continuous Profiler with SSI can be enabled for the following languages:
...
+    clusterAgent:
+      image:
+        tag: latest
+      env:
+        - name: DD_ADMISSION_CONTROLLER_AUTO_INSTRUMENTATION_PROFILING_ENABLED
+          value: "auto"
+```
+
+Image tag "latest" is used to install a recent agent version with support for profiling as Datadog
+Operator defaults to an older agent version.
```
Comment (urseberry) [inline content/en/profiler/enabling/ssi.md:95]:
```suggestion
Use the "latest" image tag to install a recent Agent version with support for profiling, as 
the Datadog Operator defaults to an older Agent version.
```

File: content/en/profiler/enabling/ssi.md [hunk 2]
```diff
@@ -26,40 +25,104 @@ Continuous Profiler with SSI can be enabled for the following languages:
...
+{{% /tab %}}
+{{< /tabs >}}
+
 ## How does profiling work with SSI
 
-After the installation, all new processes on the host or in the container are executed with the
-`DD_PROFILING_ENABLED=auto` environment variable. Running processes will not be affected.
-The Datadog library dynamically turns on the profiler for the processes that are good profiling candidates.
+After the installation, all new processes are executed with the `DD_PROFILING_ENABLED=auto`
+environment variable. Running processes will not be affected. The Datadog library dynamically turns
```
Comment (urseberry) [inline content/en/profiler/enabling/ssi.md:104]:
```suggestion
environment variable. Running processes are not affected. The Datadog library dynamically turns
```

File: content/en/profiler/enabling/ssi.md [hunk 3]
```diff
@@ -26,40 +25,104 @@ Continuous Profiler with SSI can be enabled for the following languages:
...
+on the profiler for the processes that are good profiling candidates.
 
-The logic for identifying a process as a good candidate varies by language. For Java,
-all processes that are profiled as Java applications are usually deployed as a single Java process on a
-host. For Node and Python, profiler is only turned on if the application is running for more than 30
-seconds and has created at least one tracing span.
-SSI can also be configured to inject profiling on each and every process by using
-`DD_PROFILING_ENABLED=true`.
+The logic for identifying a process as a good candidate varies by language. For Java, all processes
+are profiled as Java applications are usually deployed as a single Java process on a host. For Node
```
Comment (urseberry) [inline content/en/profiler/enabling/ssi.md:108]:
```suggestion
The logic for identifying a process as a good candidate varies by language. For Java, all processes
are profiled, as Java applications are usually deployed as a single Java process on a host. For Node
```

File: content/en/profiler/enabling/ssi.md [hunk 4]
```diff
@@ -26,40 +25,104 @@ Continuous Profiler with SSI can be enabled for the following languages:
...
+instead of `auto`.
+
+**Note**: Datadog recommends using the `auto` setting to avoid profiling low value processes.
 
 # Reverting
 
 The [Single Step APM Instrumentation][1] page contains instructions for removing all
 instrumentation. Removing instrumentation also removes profiling.
 
 Additionally, you can disable profiling by:
```
Comment (urseberry) [inline content/en/profiler/enabling/ssi.md:122]:
```suggestion
Additionally, you can disable profiling by taking one of the following steps:
```

File: content/en/profiler/enabling/ssi.md [hunk 5]
```diff
@@ -26,40 +25,104 @@ Continuous Profiler with SSI can be enabled for the following languages:
...
-* repeating the installation instructions with `DD_PROFILING_ENABLED=false` value, or
-* removing the `DD_PROFILING_ENABLED` setting from the `/etc/environment` file on the host.
+* repeating the installation instructions using `false` value in place of `auto`, or
+* removing the `DD_PROFILING_ENABLED` setting from the `/etc/environment` file on the host, for
+  host and container deployments.
```
Comment (urseberry) [inline content/en/profiler/enabling/ssi.md:125]:
```suggestion
* Repeat the installation instructions using a `false` value in place of `auto`.
* Remove the `DD_PROFILING_ENABLED` setting from the `/etc/environment` file on the host, for host and container deployments.
```

**Review by urseberry — CHANGES_REQUESTED** [review]
Thank you for making this update! I have a few wording suggestions.

**Comment by szegedi** [comment]
@urseberry thank you for the thoughtful suggestions – I applied them all.

**Review by urseberry — APPROVED** [review]

**Comment by urseberry** [comment]
Is this ready to merge, or do you want to wait for yoannrobin to review?

**Comment by szegedi** [comment]
@urseberry I'll give Yoann some time to review.

**Comment by szegedi** [comment]
@urseberry I'd like to merge it now – thank you!

**Pull request merged**